Makefile

```
@@ -13,6 +13,10 @@ ALL_OBJ = $(ALL_CC_OBJ) $(ALL_CXX_OBJ)
 
 doc:
 	doxygen docs/Doxyfile
+	$(MAKE) -C docs html
+
+show_docs:
+	@python -c "import webbrowser; webbrowser.open_new_tab('file://$(PWD)/docs/build/html/index.html')"
```

its probably more robust to start an http server to better replicate the final gh-pages environment and avoid any CORS issues from popping up
```suggestion
	@python3 -m http.server --directory docs/build/html
```

---

Done, thanks for the suggestion!